sw/device/silicon_creator/rom_ext/doc/ownership.md

```diff
@@ -0,0 +1,542 @@
+# OpenTitan Ownership Transfer
+
+> [!NOTE]
+> This document is a draft.
+
+## Introduction
+
+This document discusses the OpenTitan Ownership Transfer protocol, including the commands, operations, data and data structures needed for ownership transfer.
+Ownership Transfer allows the owner of the chip to securely root an OpenTitan chip to their own public key infrastructure and to transfer the chip to a new owner when they no longer want or need to be the owner of the chip (e.g. reselling used equipment or transferring equipment among different security domains within the same organization).
+
+### Definitions
+
+- Owner Configuration: a data structure encoding the ownership information for a given owner.
+  The owner configuration includes an owner key, one or more application keys and various owner configuration data.
+- Owner Key: an ECDSA public/private key-pair used to authenticate the ownership block.
+- Activate Key: an ECDSA public/private key-pair used to authenticate an Activate Owner command.
+  This key is endorsed by the Owner key by virtue of being present in the owner configuration.
+- Unlock Key: an ECDSA public/private key-pair used to authenticate an Unlock Owner command.
+  This key is endorsed by the Owner key by virtue of being present in the owner configuration.
+- Ownership State: The state of a chip with respect to ownership transfer.
+  A chip may be in one of the following states: `LockedOwner`, `LockedUpdate`, `UnlockedAny`, `UnlockedEndorsed`, `LockedNone`.
+- Ownership Nonce: A 64-bit random integer used as a validation challenge for all ownership-related boot services requests.
```

Comment:
nit: must it always be random, or only unique? For example, a monotonic counter is a valid nonce in many cryptographic schemes.

Reply:
I'd go with random: the point is to be unpredictable so that a signed command cannot be replayed on the same chip or on another chip with a similar counter value or to allow pre-generating a list of signed commands with a list of would-be monotonic sequence values.